### DISCUSSION TITLE
Simulation of Hole etching

### DISCUSSION THREAD

**Discussion by jais001** [ViennaTools/ViennaPS#90] [Q&A] [answered]
Hi,

I have started using the ViennaPS for the simulation of SF6O2 hole etching. I am using Python, and followed the examples added in:
code: https://github.com/ViennaTools/ViennaPS/tree/master/examples/holeEtching

The script generates a final.vtp file for 10 seconds. As mentioned I just loaded the generated VTP file in Paraview. Still, the simulation image is different from the one in ReadMe. Is there a configuration difference for the one in Readme?
generated image:
![etch1](https://github.com/user-attachments/assets/5503ea0f-f670-46b3-86c6-a00f098dc5ca)

Image in readme:
![hole_etching](https://github.com/user-attachments/assets/ce3a173d-2384-4aa2-87af-632c951aa198)

**Comment by tobre1** [comment, marked as answer]
The SF6O2 etching model has undergone several adaptations since the images in the README were created. As a result, the parameters used in the current version might differ from those in the original images, leading to the differences you're seeing in the simulation output.

**Comment by jais001** [comment]
Hi @tobre1 Thanks for taking the time to answer my question. Your response was helpful..
